test/src/dftbp/api/mm/testcases/imp_ham/dftb_in.h2o.hsd

```diff
@@ -0,0 +1,25 @@
+Geometry = GenFormat { 
+    <<< "geo_end.gen" 
+} 
+ 
+Hamiltonian = DFTB{ 
+   MaxAngularMomentum = { 
+      H = "s" 
+      O = "p" 
+   } 
+   SCC = Yes 
+   SlaterKosterFiles = Type2FileNames{ 
+      Prefix = ./ 
+      Separator = "-" 
+      Suffix = ".skf" 
+      } 
+   } 
+Options { 
+   WriteResultsTag = Yes 
+} 
+ParserOptions { 
+   IgnoreUnprocessedNodes = Yes  
```

Comment:
This is missing a parser version, so will eventually break if the HSD input changes in the future

Reply:
Fixed